app/controllers/admin/country_languages_controller.rb

```diff
@@ -0,0 +1,47 @@
+# Admin::
+module Admin
+  class CountryLanguagesController < BaseController
```

Comment:
@iandouglas @AliSchlereth So I ended up creating this class for admins adding languages to countries, and a separate languages controller for managing just language records. 

There's still the UX thing you mentioned, with altering data, clicking "Add Language" and losing any unsaved changes, but I'm going to be more focus on getting the functionality in there, even if the UX does need improvement.

Reply:
@kolyaventuri While wrapping your controller in a module works for namespacing, I've always seen it done `Admin::CountryLanguagesController`. Putting this file in the admin sub directory is what Rails will expect from using this syntax.